x-pack/plugins/alerts/server/feature.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { i18n } from '@kbn/i18n';
+import { PluginSetupContract as FeaturesPluginSetup } from '../../features/server';
+import { INDEX_THRESHOLD_ID } from '../../alerting_builtins/server';
+import { ALERTS_FEATURE_ID } from '../common';
+
+export function registerFeature(features: FeaturesPluginSetup) {
+  features.registerFeature({
+    id: ALERTS_FEATURE_ID,
+    name: i18n.translate('xpack.alerts.featureRegistry.alertsFeatureName', {
+      defaultMessage: 'Alerts',
+    }),
+    app: [],
```

Comment:
Hmm, this is interesting. I don't think there's an intent to have "Alerts" as a feature because it would be granted under other applications instead. Only actions should have the option for users to choose "All" / "Read" / "None".

<img width="763" alt="Screen Shot 2020-06-16 at 2 30 02 PM" src="https://user-images.githubusercontent.com/3694571/84813379-e605e280-afdd-11ea-95f3-2c2a1ebf389f.png">

Reply:
@gmmorris I chatted with @mikecote today about this, and I think we want to take advantage of sub-feature privileges as part of this PR, rather than adding support for them later on. I think this will give us the UX we are looking for, and the changes required won't be too bad. I'm going to try to grab you for a quick chat tomorrow morning (my time) to explain our thinking, and see if that makes sense to you too

Reply:
Ignore my review comments here about adjusting this privilege definition for now 😄

Reply:
Larry and I caught up synchronously and decided to:
1. Remove the use of `feature` here as we don't need it (for now at least)
2. I'll open a follow up issue to address alerts appearing in management when users have no privileges to use alerts.